stdlib/date/date.flux

```diff
@@ -772,4 +772,4 @@ October = 10
 November = 11
 
 // December is a constant that represents the month of December.
-December = 12
+December = 12
```

Comment:
I think if you add a newline to the end of the file, it'll drop off the PR (which is nice).